src/services/refactors/extractMethod.ts

```diff
@@ -772,116 +758,143 @@ namespace ts.refactor.extractMethod {
             changeTracker.replaceNodeWithNodes(context.file, range.range, newNodes, { nodeSeparator: context.newLineCharacter });
         }
 
-        return {
-            scope,
-            scopeDescription: getDescriptionForScope(scope),
-            changes: changeTracker.getChanges()
-        };
+        const edits = changeTracker.getChanges();
+        const renameRange = isReadonlyArray(range.range) ? range.range[0] : range.range;
 
-        function getFirstDeclaration(type: Type): Declaration | undefined {
-            let firstDeclaration = undefined;
+        const renameFilename = renameRange.getSourceFile().fileName;
+        const renameLocation = getRenameLocation(edits, renameFilename, functionNameText);
+        return { renameFilename, renameLocation, edits };
+    }
 
-            const symbol = type.symbol;
-            if (symbol && symbol.declarations) {
-                for (const declaration of symbol.declarations) {
-                    if (firstDeclaration === undefined || declaration.pos < firstDeclaration.pos) {
-                        firstDeclaration = declaration;
-                    }
+    function getRenameLocation(edits: ReadonlyArray<FileTextChanges>, renameFilename: string, functionNameText: string): number {
+        let delta = 0;
+        for (const { fileName, textChanges } of edits) {
+            Debug.assert(fileName === renameFilename);
+            for (const change of textChanges) {
+                const { span, newText } = change;
+                // Note: We are assuming that the call expression comes before the function declaration,

```

Comment:
Can you please add `TODO (acasey)` (or `TODO (17098)`)?  This assumption will be incorrect as soon as we implement Extract Local.